core-web/libs/edit-content/src/lib/fields/dot-edit-content-tag-field/dot-edit-content-tag-field.component.ts

```diff
@@ -23,4 +28,29 @@ import { DotSelectItemDirective } from '@dotcms/ui';
 })
 export class DotEditContentTagFieldComponent {
     @Input() field: DotCMSContentTypeField;
+
+    private readonly editContentService = inject(DotEditContentService);
+    private readonly controlContainer = inject(ControlContainer);
+
+    options!: Observable<string[]>;
+
+    /**
+     * Retrieves tags based on the provided query.
+     * @param event - The AutoCompleteCompleteEvent object containing the query.
+     */
+    getTags({ query }: AutoCompleteCompleteEvent) {
+        if (query.length < 3) {
+            return;
+        }
+
+        this.options = this.editContentService.getTags(query);
+    }
```

Comment:
This endpoint gives you all the tags that exists, maybe we should get them on init and then filter using JS. This is doing a request everytime we type after 3 chars.

Reply:
getTags() is triggered on completeMethod() from p-autocomplete, and it already have a debounce time and only emit one value when the user stop the keyup

We can add a extra debounce time, but it already have that validation from the p-autocomplete component

Reply:
In that context, I think is safe to leave it as it is